### ISSUE TITLE
Snapchat Filters using OpenCV

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/Face-X#66]
Snapchat Filter using OpenCV

![image](https://user-images.githubusercontent.com/57909583/103230588-d50a3680-495b-11eb-8f8f-a99d8ad20231.png)
![image](https://user-images.githubusercontent.com/57909583/103230821-624d8b00-495c-11eb-8067-728243e8bed3.png)

A very fun issue to contribute.

We will create our own filters on our facial images.

This issue will not be closed so many pull request. are welcomed.

I hope we will get some different filter in every new pr.

This issue can be assigned to many.

**Comment by Defcon27** [comment]
I am a SWOC participant, so can I work on this issue

**Comment by ritikchinu** [comment]
I want to work on this, so if you can assign it to me

**Comment by akshitagupta15june** [comment]
ok assigning to u both

**Comment by RaghavModi** [comment]
Hello, I am a SWOC participant. I would like to contribute to this project.
Can you assign me this issue?

**Comment by akshitagupta15june** [comment]
yeah sure

**Comment by koolgax99** [comment]
Hello, I am an SWoC participant. Can you please assign me this issue.

**Comment by koolgax99** [comment]
hello @akshitagupta15june can you please assign me this issue.

**Comment by akshitagupta15june** [comment]
Yeah sure

**Comment by Aayush-hub** [comment]
@akshitagupta15june Mam I want to work on it, can you please assign me this?

**Comment by BlueBlaze6335** [comment]
@akshitagupta15june ma'am i want to work on this , can you assign me to this project ?!

**Comment by akshitagupta15june** [comment]
yes @BlueBlaze6335 you can work on this!
This issue is open for all no assigning required

**Comment by akshitagupta15june** [comment]
@BlueBlaze6335 have a look at already implemented filters and try some new one https://github.com/akshitagupta15june/Face-X/tree/master/Snapchat_Filters

**Comment by mehabhalodiya** [comment]
Hello @akshitagupta15june ma'am I have added a PR.
I am a JWOC mentee!
Kindly check it out!

**Comment by akshitagupta15june** [comment]
yes checking

**Comment by akshitagupta15june** [comment]
@riyalamba07 you can start working on this issue

**Comment by abhishek-iiit** [comment]
As a GSSOC21 participant,
Can, i too work on the issue @akshitagupta15june ?
If so,can you please assign me this issue...

**Comment by akshitagupta15june** [comment]
Yes ofcourse

**Comment by Pavithra2206** [comment]
hey @akshitagupta15june can i work on creating new filter as gssoc'21 participant?

**Comment by akshitagupta15june** [comment]
yes you can too work on this issue @Pavithra2206

**Comment by Komal-99** [comment]
Hi I am GSSOC participant i want to work on this issue, Kindly assign it to me 
Also please can you tell me which all filters are already added so that I don't spent time on it?

**Comment by akshitagupta15june** [comment]
Yes you can work on this

Filter implemented till now
https://github.com/akshitagupta15june/Face-X/tree/master/Snapchat_Filters

**Comment by Komal-99** [comment]
Ok thanks :)

**Comment by Sibasish-Padhy** [comment]
Hi @akshitagupta15june I am GSSOC participant 2021 and I too wanna work on this issue can u tell me which filters have added and updated till now so that I could look for the new ones??

**Comment by Hamza3226** [comment]
Hello, I am a GSSOC'21 participant and would like to contribute to this project.

**Comment by jeelnathani** [comment]
hello, I am GSSOC '21 participant and I would like to contribute here on snapchat devil filter. 
![devilFilter](https://user-images.githubusercontent.com/37493874/111021377-c4f9e300-83f1-11eb-9ce8-86f99c0bcddb.png)

**Comment by akshitagupta15june** [comment]
yes you can work on it

**Comment by sherin527** [comment]
Hi @akshitagupta15june ma'am, I'm a GSSOC'21 participant and want to contribute to this by adding another filter. Can I also work on this ma'am?

**Comment by akshitagupta15june** [comment]
yes ofcourse

**Comment by akshitagupta15june** [comment]
yes anyone can contribute to this issue, This issue is open for everyone.

**Comment by Komal-99** [comment]
please all add gssoc level label as i guess level4 is not their in gssoc

**Comment by rudranshsharma123** [comment]
Hello, I would love to add a few Snapchat filters. Should I wait for the assignment or should I begin working on them?

**Comment by Komal-99** [comment]
> Hello, I would love to add a few Snapchat filters. Should I wait for the assignment or should I begin working on them?

Start working it is open issue but keep in mind that you may not create already existing filter

**Comment by Abhilash249** [comment]
Hi, I would like to contribute in this issue

**Comment by akshitagupta15june** [comment]
Anyone can contribute in this issue

**Comment by Abhay-sandhu** [comment]
Hi, I would like to contribute in this issue. Please assign me .

**Comment by akshitagupta15june** [comment]
You can work on it

**Comment by hemanthraj2001** [comment]
Hello, I'd like to contribute to this project. Please put me in charge.

**Comment by sreemanjusha** [comment]
Hello Ma'am, I would like to work on this project . Please assign me Ma'am.

**Comment by varshini901** [comment]
hey there, I would like to contribute to this project. please assign me.

**Comment by Komal-99** [comment]
@hemanthraj2001 @sreemanjusha @varshini901  This is an open Issue You all can work on it But please make sure you make something different filter that has not been implemented yet. 
Also, Tell the program you are part of under which you are contributing

**Comment by abhishek7997** [comment]
Hi! I would like to address this issue as well.
Also, I am a participant of Script Winter of Code (SWOC 2021).

**Comment by akshitagupta15june** [comment]
yes u can

**Comment by WhiteWolf47** [comment]
@akshitagupta15june  Is there an existing file for this or do we have to make it from scratch?

**Comment by WhiteWolf47** [comment]
@akshitagupta15june i am participating in GSSOC'22, can i work on this ?

**Comment by Sharanya-Prabhu** [comment]
This issue sounds so much fun! Can I please work on this? I'm a GSSOC'22 participant.

**Comment by akshitagupta15june** [comment]
@WhiteWolf47 @Sharanya-Prabhu you all can work on this issue and this issue can be worked on by many participants but make sure to implement different filters which are not present in the repo. So please check the Snapchat filter folder once before implementing.

**Comment by harshitbansall** [comment]
Hey can I contribute to this issue under GSSoC’2022 @akshitagupta15june .

**Comment by akshitagupta15june** [comment]
yes you can @harshitbansall

**Comment by akshitagupta15june** [comment]
Anyone can work on this issue, no assigning of participants required!

**Comment by shreya024** [comment]
Hi @akshitagupta15june I would like to work on this issue

**Comment by akshitagupta15june** [comment]
u can work on it

**Comment by Teju-g49** [comment]
please assign me issue i want to work on it

**Comment by akshitagupta15june** [comment]
You can work on it

**Comment by Sumedha4** [comment]
@akshitagupta15june  hey i would like to work on this

**Comment by Teju-g49** [comment]
How much time you are giving to me???

**Comment by Aarthy153** [comment]
I want to add a filter called Obama- which converts photos into a poster-like look. Can you assign this to me? @akshitagupta15june

**Comment by Sumedha4** [comment]
Hello i would like to contribute to this repo under Hacktoberfest. Please assign this to me

**Comment by eziorobinhood** [comment]
Hai, I would like to contribute to this repo for hacktoberfest. Can you please assign this to me.

**Comment by KuberaPrijesh** [comment]
I want to add some filters cowboy, convert face without mask into with mask and thug life filter, - which converts photos into a man with a cowboy hat, and(thug life) filter which put a cool glass on face and some colours in the background. Can you assign this to me? I want to work on this for Hacktoberfest. @akshitagupta15june

**Comment by KuberaPrijesh** [comment]
Hi @akshitagupta15june

**Comment by Soumadeep03** [comment]
Please can you assign me this project?

**Comment by akshitagupta15june** [comment]
anyone can work on it

**Comment by amelia2802** [comment]
Hello, @akshitagupta15june I want to work on this issue. Can you assign me?

**Comment by akshitagupta15june** [comment]
Ok @amelia2802 you can go for it

**Comment by akshitagupta15june** [comment]
Anyone can work on this issue there is no need of assigning anyone
